1209/‭BE7A‬/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: AXIOM Beta
+owner: apertus°
+license: GNU GPL V2 & Cern OHL
+site: https://www.apertus.org/axiom-beta
+source: https://github.com/apertus-open-source-cinema/axiom-beta-firmware & https://wiki.apertus.org/index.php/AXIOM_Beta/PCBs
```

Comment:
Please supply a single URL - this is automatically converted to a link, so mutliple URLs won't work.

Reply:
Thanks!

Understood, I updated the PR accordingly.